src/Nethermind/Nethermind.Optimism.Test/OptimismHeaderValidatorTests.cs

```diff
@@ -35,7 +34,9 @@ public class OptimismHeaderValidatorTests
         yield return ("0xffffaaaa", false);
         yield return ("0x01ffffffff00000000", false);
         yield return ("0xff0000000100000001", false);
+        yield return ("0x000000000000000000", false);
```

Comment:
In `ExtraData`, a `0` EIP-1559 parameters should not be valid, instead defaults should be used (which are non-zero)